docs/chapters/04_features.md

```diff
@@ -773,6 +773,36 @@ export class GetProductsCount {
 
 
 
+#### Using pagination
+
+The current Booster GraphQL API includes an alternative type for your read models that stands for `List{"your-read-model-name"}`, which includes the functionality to work with pagination.
+
+The Read Model List type includes some new parameters that can be used on queries:
+ * `limit`; an integer that specifies the maximum number of items to be returned.
+ * `afterCursor`; an object that includes the key of the last retrieved item, so the next query will return the next page of results.
```

Comment:
This is not accurate and it could confuse the user, as the they should not use the key of the read model, but the cursor returned in the previous query. 

The cursor should be opaque, the user should not need to know its internals, as it can change from one provider to another (this is commented in the GraphQL documentation too)

So we should say here that in `afterCursor` we should put the result of what was returned in `cursor` in the previous query (if not null)